monitoring/sensu_check.py

```diff
@@ -257,6 +263,36 @@ def sensu_check(module, path, name, state='present', backup=False):
                     changed = True
                     reasons.append('`{opt}\' was removed'.format(opt=opt))
 
+        if module.params['custom']:
+          # Convert to json
+          try:
+            custom_params = ast.literal_eval(module.params['custom'])
+          except:
+            msg = 'Module parameter "custom" contains invalid JSON. Example: custom=\'{"JSON": "here"}\''
+            module.fail_json(msg=msg)
+
+          overwrited_fields = set(custom_params.keys()) & set(simple_opts + ['type','subdue'])
+          if overwrited_fields:
+            msg = 'You can\'t overwriting standard module parameters via "custom". You are trying overwrite: {of}'.format(of=list(overwrited_fields))
+            module.fail_json(msg=msg)
+
+          for k,v in custom_params.items():
+            if k in config['checks'][name].keys():
```

Comment:
The `in` operator checks the keys when given a dict, so there\s no need for `.keys()`